components/Frame/Popover/UserNav.js

```diff
@@ -73,10 +72,10 @@ const UserNav = ({
                   </div>
                 </>
               )}
-              {!isMember && (
+              {!isMember && !inNativeApp && (
                 <Button
```

Comment:
Use `Link` here for fast client-side-navigation, re add the import from `lib/routes` and then do:

```
<Link route='pledge' passHref>
  <Button {...}>{...}</Button>
</Link>
```